code/modules/mod/modules/modules_service.dm

```diff
@@ -91,3 +91,21 @@
 	REMOVE_TRAIT(mod.wearer, TRAIT_WADDLING, REF(src))
 	if(is_clown_job(mod.wearer.mind?.assigned_role))
 		mod.wearer.clear_mood_event("clownshoes")
+
+// recharging cleaner spray module
+/obj/item/mod/module/mister/cleaner
+	name = "MOD janitorial mister module"
+	desc = "An space cleaner mister, able to clean up messes quickly. Synthesizes its own supply over time (if active)."
+	device = /obj/item/reagent_containers/spray/mister/janitor
+	volume = 100
+	active_power_cost = DEFAULT_CHARGE_DRAIN
+
+/obj/item/mod/module/mister/cleaner/Initialize(mapload)
+	. = ..()
+	reagents.flags = AMOUNT_VISIBLE
```

Comment:
are you meaning to override entirely or do you want to `|=` here?

Reply:
Override